<ui version="4.0" >
 <class>KViewUsers</class>
 <widget class="QWidget" name="KViewUsers" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="NewUser" >
       <property name="text" >
        <string>New User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ModifyUser" >
       <property name="text" >
        <string>Modify User</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="NewProfile" >
       <property name="text" >
        <string>New Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ModifyProfile" >
       <property name="text" >
        <string>Modify Profile</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGObjectsList" native="1" name="List" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGObjectsList</class>
   <extends>QWidget</extends>
   <header>qgobjectslist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
